src/web/client/dal/remoteSaveProvider.ts

```diff
@@ -166,6 +166,7 @@ async function saveDataToDataverse(
             );
 
             if (!response.ok) {
+                // TODO: Remove it when multifile stabilizes.
```

Comment:
remove this one - just verified that all that we are capturing here will be captured in final catch in any case